webapp/src/store/index.js

```diff
@@ -207,10 +207,20 @@ export default new Vuex.Store({
 			if (state.activeRoom.id !== room) return
 			state.reactions = reactions
 		},
-		'api::world.updated' ({ state, commit }, { world, rooms, permissions }) {
+		'api::world.updated' ({ state, dispatch, commit }, { world, rooms, permissions }) {
+			const oldCurrentRoomModules = state.rooms.find(r => r.id === state.activeRoom.id).modules.map(m => m.type)
```

Comment:
That find seems unneccessary. `state.activeRoom` should have `modules` already, or does it not?
```suggestion
			const oldCurrentRoomModules = state.activeRoom.modules.map(m => m.type)
```